.code-samples.meilisearch.yaml

```diff
@@ -19,34 +19,34 @@ get_documents_1: |-
     .getDocuments(params: DocumentsQuery(limit: 2, filter: 'genres = action'));
 multi_search_1: |-
   await client.multiSearch(MultiSearchQuery(queries: [
-      SearchQuery(
+      IndexSearchQuery(
         query: "pooh",
         indexUid: "movies",
         limit: 5
       ),
-      SearchQuery(
+      IndexSearchQuery(
         query: "nemo",
         indexUid: "movies",
         limit: 5
       ),
-      SearchQuery(
+      IndexSearchQuery(
         query: "us",
         indexUid: "movies_ratings"
       ),
     ]));
 faceted_search_2: |-
   await client.multiSearch(MultiSearchQuery(queries: [
-      SearchQuery(
+      IndexSearchQuery(
```

Comment:
I think it's missing the query parameter

Reply:
Yep, I've opened an issue about it #321

Reply:
I've added the query params, but I think we should fix the issue in any case.